index.html

```diff
@@ -58,121 +63,140 @@ <h3>Let's connect</h3>
+            <ul class="work-categories margin-top-12 padding-0">
+              <li><span>html</span></li>
+              <li><span>css</span></li>
+              <li><span>javascript</span></li>
             </ul>
+            <button type="button" class="button button-enabled">See Project</button>
           </div>
-          <p class="work-description text-color-primary margin-top-20">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-          <ul class="work-categories margin-top-12 padding-0">
-            <li><span>html</span></li>
-            <li><span>css</span></li>
-            <li><span>javascript</span></li>
-          </ul>
-          <button type="button" class="button button-enabled">See Project</button>
         </div>
       </div>
     </section>
 
-    <section class="about-myself padding-top-114">
+    <section id="about" class="about-myself padding-top-114">
       <div class="flex about-wrap">
-        <h2 class="section-title">About myself</h2>
-        <p class="headline-profile text-color-primary margin-top-12">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-        <a href="#" class="headline-cta margin-top-12">
-          <h3>Let's connect</h3>
-        </a>
-        <ul class="social-icons margin-top-12">
-          <li>
-            <a href="#">
-              <img class="social twitter" src="images/icons/twitter-icon.png" alt="Twitter icon">
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social linkedin" src="images/icons/linkedin-icon.png" alt="Twitter icon">
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social m" src="images/icons/m-icon.png" alt="Twitter icon">
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social github" src="images/icons/github-icon.png" alt="Twitter icon">
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social f" src="images/icons/f-icon.png" alt="Twitter icon">
-            </a>
-          </li>
-        </ul>
-        <button type="button" class="button button-enabled">See my resume</button>
+        <div class="about-details-wrap">
+          <h2 class="section-title">About <br> myself</h2>
+          <p class="headline-profile text-color-primary margin-top-12">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <a href="#" class="headline-cta margin-top-12">
```

Comment:
- [ ] Please remove the left margin from the language icons section and make sure it matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/165696862-4c487a59-18ff-4a20-b053-fea4b681a443.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165696838-b341dfd8-8469-49dd-bc00-204425117caa.png)